src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -75,6 +80,17 @@ const ProjectInstallPlatform = createReactClass({
     return location.href.indexOf('getting-started') > 0;
   },
 
+  inInstallExperiment() {
```

Comment:
I believe think there's any need for this to go on state at all - it should probably just be a method that returns a boolean and gets called in render